oneargopy/Argo.py

```diff
@@ -276,7 +287,17 @@ def trajectories(self, floats: int | list | dict)-> None:
         plt.tight_layout()
 
         # Displaying graph
-        plt.show()
+        if visible == True:
+            plt.show()
+
+        # Saving Graph
```

Comment:
This block needs to come before plt.show()

At least on Linux, plt.show() halts the execution until I close that window. At that point, plt is gone and the saved output is a small (2 KB) file, not the actual plot.